crates/iceberg/src/inspect/metadata_table.rs

```diff
@@ -59,12 +70,14 @@ pub mod tests {
     ///   or use rust-analyzer (see [video](https://github.com/rust-analyzer/expect-test)).
     ///   Check the doc of [`expect_test`] for more details.
     /// - `ignore_check_columns`: Some columns are not stable, so we can skip them.
+    /// - `ignore_check_struct_fields`: Same as `ignore_check_columns` but for (top-level) struct fields.
```

Comment:
Have to introduce this and the logic below so we can ignore `file_path` which is nested under the `data_file` struct.